UKHomeOffice/sgmr-service
# SPCR-415 Fix pleasure craft form pre-filling

### AC / Description of changes
During the voyage plan when you add a PC then add a second PC, the details of the previously added PC appear in the form. These details should not pre-fill the form.

### To Test
- Delete all PCs
- Create a voyage plan
- On PC page, add a PC
- On ‘Will you sail?’ component, click no
- > It will take you to the PC form but with the details should **not** be visible
- Select a PC 
- Continue through the form
- > Everything should submit

### Notes
There is currently a bug where you cannot select 'neither of the above' to the Charter or hire question so when filling in the PC for select either 'charter' or 'hire'

---

**Remember to merge to feature branches not `master`.**
